data/help/anaconda-gui.json

```diff
@@ -0,0 +1,85 @@
+{
+  "_comment_": [
+    "This is a mapping of screen ids and help files for Anaconda GUI on RHEL."
```

Comment:
@M4rtinK, @VladimirSlavik, I have tested the code with this mapping file and the `anaconda-user-help-9.0.1-1` scratch build. It seems to work fine. The files `anaconda-help-anchors.json`, `rhel_help_placeholder.txt` and `rhel_help_placeholder.xml` should be removed.